css/properties/position.json

```diff
@@ -73,7 +73,8 @@
                 "notes": "Before Firefox 44, <code>position: fixed</code> didn't create a stacking context in most cases. Firefox and the specification have been modified to mimic Chrome and Safari's long-time behavior."
               },
               "firefox_android": {
-                "version_added": null
+                "version_added": "4",
```

Comment:
Changes to this file conflict with https://github.com/mdn/browser-compat-data/pull/3971 So maybe leave them out here?